api/client/samples/similar_regions/similar_region_state.py

```diff
@@ -19,9 +19,10 @@ class SimilarRegionState(object):
     Holds and initializes parameters and provide ssaving/loading logic for a similar_region search.
     """
 
-    def __init__(self, region_properties, regions_to_compare, client, data_dir=None):
+    def __init__(self, region_properties, regions_to_compare, client, data_dir=None, no_download=False):
         self.client = client
         self._logger = api.client.lib.get_default_logger()
+        self.no_download = no_download
```

Comment:
rather than saving it as an instance variable, just use it directly as an argument to load()
besides reducing the state, it also makes sense to have load takea  boolean which controls whether it will download or not